data/campaigns/Heir_To_The_Throne/overworld/00a_western_wesnoth.cfg

```diff
@@ -0,0 +1,1625 @@
+            message= _ "Her and some of her girls are still down there, trapped — or dead. If’n you can help, we’d all be in yer debt.
+
+If not... well, there’s our port. I’ve got some ships to take ye back to the mainland. We owe you that much."
+        [/message]
+    )
+    (COMPLETED_MOVETO=
+    )
+}
+#######################################################################################################################################################
+#                                                               S08b - ISLE OF THE DAMNED, PART 2
+#######################################################################################################################################################
+{PLACE_POI 2 31 s08b
+    (CONDITION={VARIABLE_CONDITIONAL status_s08a equals completed}) # only show this POI if we've completed part 1
+
+    SCENARIO_FILE=08b_Isle_of_the_Damned_part2
+    (PREVIEW_WML=
+        title=_"Isle of the Damned II"
+        preview=bigmap/preview-shroud.png
+        difficulty=1
+        gold=1
+        recruit1=units/human-outlaws/poacher.png
+        companion1="units/human-outlaws/fugitive+female.png"
+    )
+    #############################
+    # INCOMPLETE
+    #############################
+    (INCOMPLETE_PRESTART=
+    )
+    (INCOMPLETE_APPROACHED=
+    )
+    (INCOMPLETE_MOVETO=
+        {MODIFY_UNIT id=Delurin facing sw}
+        [message]
+            speaker=Delurin
+            message= _ "Our leader an’ some o’ her hunters are still down there undergroud, trapped — or dead. If’n you can help, we’d all be in yer debt."
+        [/message]
+    )
+
+    #############################
+    # COMPLETED
+    #############################
+    (COMPLETED_PRESTART=
+        [remove_item]
+            image=bigmap/poi-completed.png # otherwise we have 2 "completed" indicators right next to each-other, which looks funny
+            x,y=2,31
+        [/remove_item]
+    )
+    (JUSTCOMPLETED_START=
+        {RECALL_XY Harper 3 33}
+        {MODIFY_UNIT id=Harper facing sw}
+        [message]
+            speaker=Harper
+            message= _ "...a prince? No kiddin’. Never thought I’d be friends with blue blood."
+        [/message]
+        [message]
+            speaker=Harper
+            message= _ "Well. Listen, if you’re fightin’ Asheviere, then me an’ my hunters are with you; we want to help. It’s about time someone did somethin’ ’bout her."
+        [/message]
+        {NEW_RECRUIT1 Poacher _"Konrad can now recruit Poachers!" "human-outlaws/poacher.png"}
+        {MOVE_UNDER_KONRAD( {MOVE_UNIT id=Harper $stored_konrad.x $stored_konrad.y} {PUT_TO_RECALL_LIST id=Harper} )}
+        {FIRE_EVENT_UNIT moveto id=Konrad} # trigger COMPLETED_MOVETO
+    )
+    (COMPLETED_MOVETO=
+        [message]
+            speaker=Delurin
+            message= _ "Thank you again fer how much help you’ve been, Konrad! The Three Sisters’re in your debt, now an’ ferever. Go an’ give Asheviere hell."
```

Comment:
'give them hell' sounds like a modern expression to me... but I'm not sure what could replace it.